.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "tests/ols-docker"]
+	path = tests/ols-docker
+	url = https://github.com/radome/OLS-docker.git
```

Comment:
Could you ask Thomas to set this up (having your OLS-docker git in Ensembl workspace)

Reply:
@marcoooo @radome: I've created the repo here: https://github.com/Ensembl/OLS-docker

Reply:
> @marcoooo @radome: I've created the repo here: https://github.com/Ensembl/OLS-docker

I've created a [PR](https://github.com/Ensembl/OLS-docker/pull/1) for that.

Reply:
PR validated.